src/sentry/models/integrationfeature.py

```diff
@@ -31,10 +37,16 @@ def as_str(cls, feature):
             return "integrations-stacktrace-link"
         elif feature == cls.EVENT_HOOKS:
             return "integrations-event-hooks"
+        elif feature == cls.PROJECT_MANAGEMENT:
+            return "integrations-project-managment"
+        elif feature == cls.INCIDENT_MANAGEMENT:
+            return "integrations-incident-management"
+        elif feature == cls.FEATURE_FLAG:
+            return "integrations-feature-flag"
 
     @classmethod
     def description(cls, feature, name):
-        if feature == cls.API:
+        if feature in [cls.API, cls.PROJECT_MANAGEMENT, cls.INCIDENT_MANAGEMENT, cls.FEATURE_FLAG]:
```

Comment:
@NisanthanNanthakumar This fine for now but note that I will update this text in an upcoming PR so they aren't all the same